react-app/src/features/faq/content/chpRenderSection.tsx

```diff
@@ -0,0 +1,42 @@
+
+interface Template {
+    title: string;
+    text: string;
+    href: string;
+    subtext?: string[];
+  }
+  
+  export const renderSection = (
+    title: string,
+    templates: Template[],
+    filterCondition: (template: Template) => boolean,
+    ulClassName: string = "" 
+  ): JSX.Element => (
+    <>
+      <p>{title}</p>
+      <ul className={`list-disc pl-6 space-y-2 ${ulClassName}`}>
```

Comment:
Can you use the `cn` utility function instead? 

```tsx
<ul className={cn('list-disc pl-6 space-y-2', ulClassName)}>
```

Reply:
Happy to explain over Slack what it does, but [this article](https://tigerabrodi.blog/the-story-behind-tailwinds-cn-function) will do a lot better

Reply:
done!